ChainLink-Server/graphql/resolvers/users.js

```diff
@@ -124,21 +125,70 @@ async function refreshStravaToken(username, refreshToken) {
 }
 module.exports = {
   Query: {
+    async getUser(_, { username, userID }) {
```

Comment:
this can go away now right? since they are separated below

Reply:
Whoops, thanks or catching that. I had some messing branching issues. Corrected